### PR TITLE
Write button & icon stories; change to variant & appearance

### PR DESCRIPTION
**Pull request by tobiaslins**

### CODE REVIEW

File: web/src/components/icon/index.tsx [hunk 1]
```diff
@@ -7,12 +7,13 @@ import * as icons from '../../icons'
 type IconSize = 'small' | 'medium' | 'large'
 
 export type IconTypes = 'arrowLeft' | 'close' | 'people' | 'person'
-type GenericIconProps = Omit<DOMProps & SVGAttributes<SVGElement>, 'name'> & {
+export type GenericIconProps = Omit<DOMProps, 'name'> & {
```
Comment (AleshaOleg) [inline web/src/components/icon/index.tsx:10]: Might be `IconProps` instead `GenericIconProps`?

**Comment by AleshaOleg** [comment]
Please add designs to each story

File: web/src/stories/button/base.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+export default {
+  title: 'Components/Button',
+  component: Button,
+  argTypes: { onPress: { action: 'clicked' }, children: { control: 'text' } },
+}
+
+const Template = props => {
+  return <Button {...props} />
+}
+export const Base = Template.bind({})
```
Comment (AleshaOleg) [inline web/src/stories/button/base.stories.tsx:13]: Why we need `.bind` here?

**Comment by AleshaOleg** [comment]
Can we do something with?

```
/* eslint-disable import/no-unresolved */
import React, { SVGAttributes } from 'react'
import { DOMProps } from '@react-types/shared'
```
in `components/icon/index.tsx`

It's already in code, but would be good to fix before merging this one.

File: web/src/components/icon/index.tsx [hunk 3]
```diff
@@ -35,9 +36,16 @@ export const Icon: React.FC<GenericIconProps> = ({
   ...rest
 }) => {
   const IconComponent = getFromSize(name, size)
-  return <IconComponent style={{ color }} {...rest} />
+  return (
+    <IconComponent
+      style={{ color }}
+      {...rest}
+      {...(rest['aria-label'] ? { 'aria-hidden': false } : {})}
```
[the remark this answers is not in the document]
Comment (tobiaslins) [reply web/src/components/icon/index.tsx]: done in 6e65384799c03f69cf3a68d6c32b422fe645aa28

File: web/src/components/icon/index.tsx (hunk 1 above)
Comment (tobiaslins) [reply web/src/components/icon/index.tsx:10]: true, was some old code that I had. thanks!

Comment (tobiaslins) [reply web/src/components/icon/index.tsx]: fixed in b532f29240724a50183701ae024affcc799bc705

**Comment by tobiaslins** [comment]
> Can we do something with?
> 
> ```
> /* eslint-disable import/no-unresolved */
> import React, { SVGAttributes } from 'react'
> import { DOMProps } from '@react-types/shared'
> ```
> 
> in `components/icon/index.tsx`
> 
> It's already in code, but would be good to fix before merging this one.

done!

**Comment by tobiaslins** [comment]
> Please add designs to each story

done!

**Review by AleshaOleg — APPROVED** [review]

**Pull request merged**